packages/renderer/src/lib/kube/details/KubePortsList.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+import Cell from '/@/lib/details/DetailsCell.svelte';
+import { kubernetesCurrentContextPortForwards } from '/@/stores/kubernetes-contexts-state';
+import type { type UserForwardConfig, WorkloadKind } from '/@api/kubernetes-port-forward-model';
```

Comment:
`type` inside `{}` are not necessary?

Reply:
Yes, prettier got crazy, this does not even transpile, fixed